src/test/jtx/impl/flags.cpp

```diff
@@ -51,7 +51,9 @@ void
 nflags::operator()(Env& env) const
 {
     auto const sle = env.le(account_);
```

Comment:
The data members `flags::account_` and `nflags::account_` always hold a valid instance of the `Account` class. These data members are not pointers or `std::optional` values.

Hence, `env.le(account_)`  must always exist, isn't it? Are you envisioning a future modification to these `flags` and `nflags` classes, due to which a null pointer might be returned here?

Reply:
A valid instance of an `Account` does not guarantee that that account has been created / funded on the ledger. So even though the `Account` is valid, a null `sle` could easily be returned. Consider this test:
```
Env env{*this};
Account alice("alice");

env.require(flags(alice, asfRequireDest));
env.require(nflags(alice, asfRequireAuth));
```
That's a perfectly validly written test, but without this change, it'll dereference a null `SLE` and crash. With this change, the test will fail in the changed functions, but it won't crash.

Reply:
okay, thanks for the clarification.